### DISCUSSION TITLE
Problème avec les altitudes en mode "Managed"

### DISCUSSION THREAD

**Discussion by kampinod** [flybywiresim/aircraft#2958] [Help]
Bonjour à tous,
J'ai un souci avec le mode "Managed" de A32NX, à priori, (ou est-ce une coincidence?) depuis la dernière version 1.12.13.0 l'avion ne respecte plus les "Altitudes" du Plan de vol du MCDU/F-PLN, route et vitesse sont bien respectées, mais lorsqu'on clique sur le haut du bouton "ALT" pour passer en mode "Managed", l'avion continue à suivre l'altitude qui s'affiche sous ce bouton au lieu de suivre les "Altitudes managées" du plan de vol...J'ai fait quelques essais avant l'arrivée de la version 1.12.13.0, qui semblaient bien fonctionner....
Par ailleurs pour information, j'utilise toujours  la version Windows 10 / 1909, alors que vous semblez préconisez la version 2004...mais j'hésite beaucoup à l'installer compte tenu des problèmes que j'ai pu lire sur le web sur le sujet...?
Auriez-vous la gentillesse d'éclairer ma lanterne...?
Un grand merci à la communauté pour cet avion qui s'approche toujours plus de la réalité...

**Comment by nistei** [comment]
@Benjozork

**Comment by Inaxair** [comment]
Bonsoir!

Tout d'abord, la version 2004 de Windows 10 est la version recommandé par Microsoft pour le simulateur. Si on parle des problèmes, personellement j'en ai aucun. Concernant le mode managed, il faut s'assurer que l'altitude soit bien rentrée dans le plan de vol. Il devrait avoir un ____/FL (Altitude) concernant l'altitude. Il faudrait s'assurer que ce soit bien rentrée, rerentrer au besoin.

Essayez àa, puis si ça ne fonctionne pas, on va essayer d'autres résolutions.

Merci beaucoup!
Inaxair de FlyByWire Simulation

**Comment by kampinod** [comment]
Bonsoir,
Tout d'abord, un grand merci pour votre réactivité et votre réponse,
Je vérifie à chaque vol que l'altitude est bien renseignée dans le plan de vol (MCDU/F-PLN), j'ai même renseigné des Altitudes "Contraintes", mais même ces altitudes ne sont pas respectées, bien qu'elles s'affichent bien en mauve sur le ND...
Ce qui est curieux c'est que à priori cela fonctionnait normalement avant la mise à jour 1.12.13.0 (à moins que ce ne soit qu'une coincidence?) mais c'est ce que j'ai constaté...
J'ai visionné plusieurs fois le tuto de Sake Simu https://www.youtube.com/watch?v=wWILEYJDVuU
et j'ai appliqué ses conseils, et cela fonctionnait bien jusqu'à l'arrivée de la nouvelle version...
Je me suis inscrit sur Github alors que je n'ai aucune expérience dans le développement, donc navré de ne pouvoir vous apporter ma contribution, néanmoins dans un esprit constructif, j'essaierai de faire remonter les infos concernant les éventuels bugs rencontrés dans l'espoir que cela aidera l'équipe d'améliorer encore ce beau projet... Je suis en retraite et j'utilise FS chaque jour et suis dispo pour apporter ma petite pierre à l'édifice.
Bonne soirée à vous et encore merci pour tout ce travail...
J.C. SIMON
[https://www.bing.com/th?id=OVP.a3vWtD2r9pwafua7IjE6owEsDh&pid=Api]<https://www.youtube.com/watch?v=wWILEYJDVuU>
FS2020 | Tuto complet A320 [2/2] : Navigation, décollage et atterrissage<https://www.youtube.com/watch?v=wWILEYJDVuU>
Bienvenue dans cette nouvelle vidéo tuto complet où nous ré-abordons les fondamentaux du pilotage en liner sur FS2020. Dans cette deuxième partie, nous verrons ensemble comment manier l'A320, du décollage à l'atterrissage. Serveur Discord de la chaîne : https://discord.gg/dhH2E3R Lien de téléchargement de la livrée : https://flightsim ...
www.youtube.com

________________________________
De : Inaxair <notifications@github.com>
Envoyé : dimanche 10 janvier 2021 00:02
À : flybywiresim/a32nx <a32nx@noreply.github.com>
Cc : kampinod <jacsimon@live.fr>; Author <author@noreply.github.com>
Objet : Re: [flybywiresim/a32nx] PROBLEME AVEC LES ALTITUDES EN MODE MANAGED (#2958)


Bonsoir!

Tout d'abord, la version 2004 de Windows 10 est la version recommandé par Microsoft pour le simulateur. Si on parle des problèmes, personellement j'en ai aucun. Concernant le mode managed, il faut s'assurer que l'altitude soit bien rentrée dans le plan de vol. Il devrait avoir un ____/FL (Altitude) concernant l'altitude. Il faudrait s'assurer que ce soit bien rentrée, rerentrer au besoin.

Essayez àa, puis si ça ne fonctionne pas, on va essayer d'autres résolutions.

Merci beaucoup!
Inaxair de FlyByWire Simulation

**Reply by Inaxair** [reply]
Bonsoir!

Nous travaillons sur un nouveau système de pilote automatique et un nouveau système de contrôle de plan de vol, donc ceci devrait alors être résolu dès que nous allons pouvoir ajouter ce système.

Meilleurs voeux,
Inaxair de FlyByWire Simulations

**Comment by kampinod** [comment]
Grand merci à vous et bon courage...
Bonne journée
J.C. SIMON
________________________________
De : Inaxair <notifications@github.com>
Envoyé : jeudi 14 janvier 2021 04:04
À : flybywiresim/a32nx <a32nx@noreply.github.com>
Cc : kampinod <jacsimon@live.fr>; Author <author@noreply.github.com>
Objet : Re: [flybywiresim/a32nx] Problème avec les altitudes en mode "Managed" (#2958)


Bonsoir!

Nous travaillons sur un nouveau système de pilote automatique et un nouveau système de contrôle de plan de vol, donc ceci devrait alors être résolu dès que nous allons pouvoir ajouter ce système.

Meilleurs voeux,
Inaxair de FlyByWire Simulations

**Comment by kampinod** [comment]
Bonsoir,
Auriez-vous la gentillesse de me dire, comment identifier la version A32NX qui est dans le dossier "Community" après l'avoir téléchargée avec "A32NX Downloader"?
Mille mercis et bon week-end
J.C. SIMON
________________________________
De : Inaxair <notifications@github.com>
Envoyé : jeudi 14 janvier 2021 04:04
À : flybywiresim/a32nx <a32nx@noreply.github.com>
Cc : kampinod <jacsimon@live.fr>; Author <author@noreply.github.com>
Objet : Re: [flybywiresim/a32nx] Problème avec les altitudes en mode "Managed" (#2958)


Bonsoir!

Nous travaillons sur un nouveau système de pilote automatique et un nouveau système de contrôle de plan de vol, donc ceci devrait alors être résolu dès que nous allons pouvoir ajouter ce système.

Meilleurs voeux,
Inaxair de FlyByWire Simulations

**Reply by kampinod** [reply]
Bonjour,
Je reviens vers vous concernant le mauvais fonctionnement que je constate sur A32NX, (mon message du 9/1/21), Je suis passé depuis qqs jours à la version Windows 10-2004 ainsi que vous le recommandez, mais le problème reste toujours le même, l'avion ne suit pas le plan de vol du MCDU (Alt. vitesses rarement respectées, bien que la bille jaune soit affichée, seule les routes sont conformes au F-PLN...) Pourriez-vous me dire si d'autres simmers ont constaté ce problème? Ou bien si des corrections sont en cours...? En tout cas, un Grand merci pour votre implication...Bonne journée
